templates/view.html

```diff
@@ -98,12 +97,7 @@
       <div class="pastey-title">
         <h1 class="mb-3 h2" id="pastey-title">{{ paste.title }}</h1>
       </div>
-      
-      <span style="font-weight:bold;">Format: </span>{{ paste.language }}</span>
-      {% if not paste.encrypted %}
```

Comment:
Hi, thank you for all of the work recently. It seems you were making these changes against an older version of the code though, as this should not be removed. Please rebase your branch to make use of the latest version if possible.

Reply:
I think I fixed it. The latest commit on this branch should be correct.